### DISCUSSION TITLE
Uploading by windows - No title just says Indoor Cycling?

### DISCUSSION THREAD

**Discussion by kjillman** [philosowaffle/peloton-to-garmin#437] [Q&A]
Apologies in advance, think I'm being thick! I'm uploading via windows and everything uploads correctly , apart from the Title of the ride (45_min_Power_Zone_Endurance_EDM_with_Ben_Alldis) it just says indoor cycling!

I must be doing something wrong, help would be appreciated.

Thanks  
Keith

**Comment by philosowaffle** [comment]
In your `configuration.local.json` file what do you have set for [Garmin.FormatToUpload](https://philosowaffle.github.io/peloton-to-garmin/configuration/json.html#garmin-config)?

**Reply by kjillman** [reply]
"Fit"

**Reply by philosowaffle** [reply]
Probably going to have to share some logs and/or your config file.  Usually when the workout title is missing, that indicates that a TCX file is being uploaded.  So something between the config file and what P2G is using isn't squared away.

**Reply by kjillman** [reply]
Hi, 

Thanks for looking into this, my config file is below

![image](https://user-images.githubusercontent.com/85358145/216827683-4decac56-bd58-4adc-a8c7-a83ab9cb1e1f.png)

**Reply by philosowaffle** [reply]
Config looks good. In your logs, do you see a line similar to the below:
```
[09:57:03 INF] Uploading to Garmin: G:\Code\Git\peloton-to-garmin\src\PelotonToGarminConsole\working\upload\4833826440dc49b9a20ec234ee2db777_5_min_Breathing_Meditation_with_Ross_Rayburn.fit
```

Most importantly, we're looking to see if the `Uploading to Garmin:` log line has `.fit` at the very end of the line.

**Reply by kjillman** [reply]
yes I do

![image](https://user-images.githubusercontent.com/85358145/216832259-a96cbef0-2c6f-4ede-9734-06523da7d157.png)

**Comment by kjillman** [comment]
Any ideas why this isn't working correctly for me?

**Reply by philosowaffle** [reply]
Unfortunately no. Everything looks correct on P2G's end (judging purely by the screenshots above).  And I'm not able to reproduce any issue. 

The only other thing I can think of is if you're also recording these workouts on your watch? If your watch record is getting synced to Garmin, that could block P2G from uploading its version of the workout, or you could also end up with duplicate workouts one that has a title and one that says "Indoor Cycling".
